### DISCUSSION TITLE
Child of child in navbar

### DISCUSSION THREAD

**Discussion by aewingate** [CollectionBuilder/collectionbuilder.github.io#61] [Q&A]
Hi!

Is it possible to create a child of a child in the navigation bar? (i.e., an item in the dropdown has it's own dropdown?)

In my site here (https://aewingate.github.io/cb_demo/), I'd like to be able to have three options (Series History, Characteristics, Other Issues) under each of La Novela Corta, La Novela Teatral, and La Novela Comica which are themselves under Explore the Series. 

This is what I currently have in my config-nav.csv (I guessed at a possible solution), but it's not working:
![image](https://user-images.githubusercontent.com/72820848/196988042-8b165c3a-2777-4cee-863f-d2f804d4917f.png)

**Comment by evanwill** [comment]
@aewingate that type of nesting isn't currently supported by the template because it isn't possible in standard Bootstrap--> the nav is created using the standard [Bootstrap navbar component](https://getbootstrap.com/docs/5.2/components/navbar/). The navbar supports [Bootstrap dropdowns](https://getbootstrap.com/docs/5.2/components/dropdowns/) as a nav item, however, dropdowns do not support nesting another dropdown inside. So the arrangement you describe can't be done using Bootstrap dropdowns out of the box.

If you search around, there are some examples of methods to implement multi-level bootstrap dropdowns by adding some additional CSS and JS--in most cases these solutions don't really work on mobile in conjunction with the navbar (at least I haven't seen a working example!). So I believe to get it to work, it would be better to build a fully customized nav not using the standard Bootstrap navbar. Basically you would customize the file "_includes/collection-nav.html" and add some additional CSS in "_sass/_custom.scss". 

If that sounds like too much customization, I would consider simplifying main nav structure and having the subnav on the relevant page(s). Having too many options exposed in the top nav can make it more confusing for users at times, since there is several clicks necessary to get anywhere. 
For example, you could keep "Explore the Series" dropdown with La Novela Corta, La Novela Teatral, and La Novela Comica. Then on each of those pages you have a nav at the top of the page with the options Series History, Characteristics, Other Issues (those could be sections on the page or separate pages).

**Comment by aewingate** [comment]
This is all great to know, thank you so much for such detailed information!
For context, I'm the TA for the digital libraries course at IU that is using CollectionBuilder, so this is good information to have in case my students ask!

**Reply by evanwill** [reply]
@aewingate awesome! Please let us know how it goes and send along questions!
